### ISSUE TITLE
InvalidOperationException: You must call the Bind method before performing this operation.

### ISSUE THREAD

**Issue by Atrejoe** [Atrejoe/SPA-Call-Manager-Pro#1]
View details in Rollbar: [https://rollbar.com/SPACallmanagerPro/SPA-Callmanager-Pro/items/3/](https://rollbar.com/SPACallmanagerPro/SPA-Callmanager-Pro/items/3/)

```
Traceback (most recent call last):
  File "System.Net.Sockets.Socket", line unknown, in ReceiveFrom(System.Byte[] buffer, System.Int32 offset, System.Int32 size, System.Net.Sockets.SocketFlags socketFlags, System.Net.EndPoint& remoteEP)
  File "System.Net.Sockets.UdpClient", line 42, in Receive(System.Net.IPEndPoint& remoteEP)
  File "D:\Users\Robert\Downloads\CiscoSource\CiscoSource\Cisco\ClsPhone.vb", line 161, in BgwUDP_DoWork(System.Object sender, System.ComponentModel.DoWorkEventArgs e)
InvalidOperationException: You must call the Bind method before performing this operation.
```

**Comment by Atrejoe** [comment]
This seems to have started since the refactor of CLSPhone (c589ff2a035a189d1391d1c971b9a5e75c309acc)

**Comment by Atrejoe** [comment]
Fixed in ef7920639673655a85f0f68637e5b4575e974530
